### PR TITLE
feat: Added ability to pass params to user defined column resolver

### PR DESCRIPTION
**Pull request by amanenk**
- added ability to set resolver params for user defined column
- removed skipping relations of same type with different names

### CODE REVIEW

File: providers/cq-provider-aws.hcl [hunk 1]
```diff
@@ -1,6 +1,6 @@
 service = "aws"
 
-output_directory = "../cq-provider-aws/resources/services/ec2"
```
Comment (roneli) [inline providers/cq-provider-aws.hcl:3]: suggest moving this to cq-provider-aws

File: codegen/definitions.go [hunk 2]
```diff
@@ -72,9 +72,6 @@ func (t TableDefinition) RelationExists(cfg config.RelationConfig) bool {
 		if rel.Name == cfg.Rename {
 			return true
 		}
-		if rel.path == cfg.Path && cfg.Name != rel.Name {
-			return true
-		}
```
Comment (roneli) [inline codegen/definitions.go:77]: why did you remove this?

**Review by roneli — COMMENTED** [review]
Had a few comments, LGTM overall.

File: codegen/definitions.go (hunk 2 above)
Comment (amanenk) [reply codegen/definitions.go:77]: this construction does not allow to have more than one relation of the same type with different names. for example if we have `ingress` rule and `eggress` rule that can have the same type but different names.

**Review by roneli — APPROVED** [review]
